app/jobs/send_accepted_rejected_notification_job.rb

```diff
@@ -0,0 +1,32 @@
+class SendAcceptedRejectedNotificationJob < ApplicationJob
```

Comment:
this job is really just to send accepted messages to 6ish clients on demo. Since its only a few people I was just planning to run it in the prod console with each individual id. 

With our messaging tracking logic, even if we run this job for someone that received this message it won't run. 

I plan on removing this job once we run it as not to junk up the jobs folder